### ISSUE TITLE
Issue in routing to the example page installation page

### ISSUE THREAD

**Issue by lamodots** [maxnowack/signaldb#354]
![image](https://github.com/maxnowack/signaldb/assets/91544048/70ef07aa-1e58-4222-ad18-bd8dc9c94801)
Fig 1

![image](https://github.com/maxnowack/signaldb/assets/91544048/f2339753-a462-4706-ab15-863a517d7d0e)
Fig 2

Hi Maxnowack , I can across signaldb yesterday when researching on client side database and I must say you have done a great work here. 
Why going through the documentation, I discovered that the link to Fig1  from Fig 2 is broken. I wish to contribute to this project and I will appreciate it if I can start from fixing the broken link.

**Comment by maxnowack** [comment]
Hey @lamodots! thanks for pointing that out. It seems that it works again, if I hard reload the page.
I think it has something to do with vitepress, as the example is a standalone nextjs app and is only copied to the vitepress public folder.
Maybe it can be fixed by changing the vitepress config ([`docs/.vitepress/config.mts`](https://github.com/maxnowack/signaldb/blob/main/docs/.vitepress/config.mts)).
Another option (but not preferred) would be to remove the "Next page" link completely on the Installation page.

You can test things out by running the following commands:
````bash
# installation of dependencies and preparing stuff
npm install && npm run docs:build

# you can run the docs in development mode with this command
npm run docs:dev
````

See also the vitepress docs: https://vitepress.dev/reference/site-config

**Comment by maxnowack** [comment]
will be fixed by vuejs/vitepress#3248

**Comment by maxnowack** [comment]
resolved by #374

**Comment by lamodots** [comment]
> Hey @lamodots! thanks for pointing that out. Feel free to contribute! It seems that it works again, if I hard reload the page. I think it has something to do with vitepress, as the example is a standalone nextjs app and is only copied to the vitepress public folder. Maybe it can be fixed by changing the vitepress config ([`docs/.vitepress/config.mts`](https://github.com/maxnowack/signaldb/blob/main/docs/.vitepress/config.mts)). Another option (but not preferred) would be to remove the "Next page" link completely on the Installation page.
> 
> You can test things out by running the following commands:
> 
> ```shell
> # installation of dependencies and preparing stuff
> npm install && npm run docs:build
> 
> # you can run the docs in development mode with this command
> npm run docs:dev
> ```
> 
> See also the vitepress docs: https://vitepress.dev/reference/site-config

Noted, thanks for the clarity.
